ipn/ipnlocal/local.go

```diff
@@ -117,6 +119,12 @@ import (
 	"tailscale.com/wgengine/wgcfg/nmcfg"
 )
 
+var metricAdvertisedRoutes expvar.Int
+
+func init() {
+	usermetric.Publish("tailscaled_advertised_routes", &metricAdvertisedRoutes)
```

Comment:
This will create a counter metric with no HELP, which I don't think is a good approach for user-facing metrics.

How about we make usermetric expose a simple gauge? It can either use expvar directly (https://github.com/tailscale/tailscale/commit/2b012731243d10c055e64c093db472c99f9b9ea7) or wrap a multilabelmap.